### ISSUE TITLE
Code not working for PHP's strict mode

### ISSUE THREAD

**Issue by oxygens** [PhenX/php-font-lib#39]
If running with highest strictness level

```
set_error_handler(function($nSeverity, $strMessage, $strFilePath, $nLineNumber){
    throw new ErrorException($strMessage, /*nExceptionCode*/ 0, $nSeverity, $strFilePath, $nLineNumber);
}, /*E_STRICT*/ -1);
```

There are places where undefined array indexes default to 0 or something.

For example:
\FontLib\BinaryStream::w($type, $data)
[...]

```
for ($i = 0; $i < $type[1]; $i++) {
    $ret += $this->w($type[0], $data[$i]);
}
```

$data[$i] doesn't always exist.

**Suggested fix:**

```
      for ($i = 0; $i < $type[1]; $i++) {
        $ret += $this->w($type[0], isset($data[$i])?$data[$i]:0);
      }
```
##

**Comment by oxygens** [comment]
Currently I am forced to set_error_handler() to a handler which ignores errors, run PHPFontLib stuff, then restore_error_handler().
Not pretty.

**Comment by PhenX** [comment]
Do you have a sample font giving this warning ? I never had this problem.

**Comment by BrunoDeBarros** [comment]
See PR #37. If you run the HTML in the PR through dompdf, you'll get that error in php-font-lib.

**Comment by oxygens** [comment]
About half the fonts (or more) in GoogleFonts if I remember correctly.

**Comment by PhenX** [comment]
Fixed by 010c3e0e22a9ab0d9f93678692c80865f041e2d2
